ui/src/components/Sidebar/SidebarRow.tsx

```diff
@@ -4,12 +4,13 @@ import styles from './styles.module.scss';
 interface Props {
   icon: string;
   text: string;
+  onClick?: () => void;
   //   isActive: boolean;
 }
 
 export const SidebarRow: React.FC<Props> = props => {
   return (
-    <div className={styles.container_row}>
+    <div className={styles.container_row} onClick={props.onClick}>
```

Comment:
`onClick` is optional as defined in the Props. `click`ing on this `div` will throw an error if `props.onClick` is null or undefined. Better to put a check for it.

```js
onClick={() => props.onClick && props.onClick()}
```